File: backend/tests/fixtures/indexing/python/sample.py
"""Sample module."""

from collections import Counter


CONSTANT = 42


def greet(name: str) -> str:
    """Return a friendly greeting."""
    return f"Hello, {name}"


class Greeter:
    def __init__(self, prefix: str) -> None:
        self.prefix = prefix

    def render(self, name: str) -> str:
        return f"{self.prefix} {greet(name)}"

File: backend/tests/fixtures/indexing/javascript/sample.js
import { readFile } from "node:fs/promises";

export class Greeter {
  constructor(prefix) {
    this.prefix = prefix;
  }

  render(name) {
    return `${this.prefix} ${name}`;
  }
}

export function greet(name) {
  return `Hello, ${name}`;
}
